source/whats-new.txt

```diff
@@ -220,3 +221,31 @@ The 2.22 driver release includes the following new features:
   information on these events, see the
   `MongoDB SDAM Logging and Monitoring specification.
   <https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring-logging-and-monitoring.rst#log-messages>`__
+
+.. _csharp-previous-versions:
+
+Previous Versions
```

Comment:
Note to tech reviewer: let me know if you think this is a good place for these links

Reply:
yeah it's good